fmatch/tests/test_matcher.py

```diff
@@ -78,7 +78,7 @@ def test_get_uuid_by_metadata(matcher_instance):
 
 
 def test_match_kube_burner(matcher_instance):
-    result = matcher_instance.match_kube_burner(["uuid1"])
+    result = matcher_instance.match_kube_burner(["uuid1"],index="ospst-*")
```

Comment:
for the test lets call out ospst-kube-burner (if that is the index) because we also have other indexes being dropped into `ospst`

Reply:
Since this is just a mockup test it really does not really matter what index I pass through.. it'll return the preset mock value instead of querying our es-instance.